WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -108,6 +108,10 @@ extension URL {
         return components.count == 4 && isHostedAtWPCom
     }
 
+    var isA8CEmoji: Bool {
```

Comment:
Random drive-by nitpicky comment :) Would something like `isWPComEmoji` be more outward facing friendly vs `isA8CEmoji`?

Reply:
Yep! Good call. 👍

Reply:
Completed in d022492b3db3540064d3c84b33831c6f6a025c7c.